src/callbacks/callbacks.jl

```diff
@@ -203,8 +207,18 @@ NVTX.@annotate function rrtmgp_model_callback!(integrator)
                 max(cloud_fraction, eps(FT))
             @. ᶜfrac = cloud_fraction
             # RRTMGP needs effective radius in microns
-            @. ᶜreliq = FT(12)
-            @. ᶜreice = FT(25)
+            @. ᶜreliq = ifelse(
+                cloud_liquid_water_content > eps(FT),
```

Comment:
I wonder if eps(FT) threshold is too large for Float32? Maybe we can use 0 here? Hopefully if doesn't matter too much when the cloud water is very small.

Reply:
Sounds good. will change that